examples/funloc/analysis_fun.py

```diff
@@ -96,9 +96,9 @@
     do_score=True,  # do scoring
     # Make SUBJ/raw_fif/SUBJ_prebad.txt file with space-separated
     # list of bad MEG channel numbers, needed for running SSS.
-    push_raw=True,  # Push raw files and SSS script to SSS workstation
-    do_sss=True,  # Run SSS remotely
-    fetch_sss=True,  # Fetch SSSed files
+    push_raw=False,  # Push raw files and SSS script to SSS workstation
+    do_sss=True,  # Run SSS remote using Maxfilter or local with mne-python
+    fetch_sss=False,  # Fetch SSSed files from SSS workstation
```

Comment:
I think if these are `True` and `sss_type == 'python'` they should just do nothing (return without pushing or fetching), so these changes can be reverted once those changes are made.